pkg/api/validation/schema.go

```diff
@@ -88,6 +88,9 @@ func (s *SwaggerSchema) ValidateBytes(data []byte) error {
 		return fmt.Errorf("kind not set")
 	}
 	allErrs := s.ValidateObject(obj, apiVersion.(string), "", apiVersion.(string)+"."+kind.(string))
+	if len(allErrs) > 0 {
+		glog.Warningf("found schema errors; if you choose to ignore them, turn validation off with --validate=false")
```

Comment:
I agree with what you said, but I used glog warnings because all the validation failure are printed as glog errors, which looks like:
```console
$ cluster/kubectl.sh create -f hello-world3-rc.yaml 
E0819 10:57:37.145443   51079 schema.go:147] Validation failed for: containers, [map[name:hello command:[/bin/echo hello, world] image:ubuntu:14.04 k8s:true]]
E0819 10:57:37.145517   51079 schema.go:147] Validation failed for: spec, map[containers:[map[command:[/bin/echo hello, world] image:ubuntu:14.04 k8s:true name:hello]] restartPolicy:OnFailure]
E0819 10:57:37.145534   51079 schema.go:147] Validation failed for: template, map[metadata:map[labels:map[name:hello-world]] spec:map[restartPolicy:OnFailure containers:[map[command:[/bin/echo hello, world] image:ubuntu:14.04 k8s:true name:hello]]]]
E0819 10:57:37.145556   51079 schema.go:147] Validation failed for: spec, map[template:map[metadata:map[labels:map[name:hello-world]] spec:map[containers:[map[command:[/bin/echo hello, world] image:ubuntu:14.04 k8s:true name:hello]] restartPolicy:OnFailure]] replicas:2 selector:map[name:hello-world]]
W0819 10:57:37.145657   51079 schema.go:92] found schema errors; if you choose to ignore them, turn validation off with --validate=false
error validating "hello-world3-rc.yaml": error validating data: found invalid field k8s for v1.Container
```

Should we change that too?